src/block/blocks/Poppy.ts

```diff
@@ -3,16 +3,14 @@ import Flowable from '../Flowable';
 
 export enum FlowerType {
     Poppy = 0,
-    BlueOrchid,
-    Allium,
-    Houstonia,
-    TulipRed,
-    TulipOrange,
-    TulipWhite,
-    TulipPink,
-    OxeyeDaisy,
-    CornFlower,
-    LilyOfTheValley
+    BlueOrchid = 1,
+    Allium = 2,
+    AzureBluet = 3,
+    RedTulip = 4,
+    OrangeTulip = 5,
+    WhiteTulip = 6,
+    PinkTulip = 7,
+    OxeyeDaisy = 8
```

Comment:
Typescript automatically increments enums, please revert this change 😄

Reply:
(forgot this file sorry 😂)

Reply:
@Bavfalcon9 Yes it does but we've been explicit about enums everywhere else so would be weird to only have it auto increments here.

Reply:
requested mainly caused enums were removed